src/components/Nav.js

```diff
@@ -16,8 +16,12 @@ class Nav extends React.Component {
         <div className="nav-1">
           <ul>
             <li className="bookstore">Bookstore&nbsp;CMS</li>
-            <li className="books"><Link to="/">Books</Link></li>
-            <li className="categories"><Link to="/categories">Categories</Link></li>
+            <li className="books">
+              <Link to="/">Books</Link>
+            </li>
+            <li className="categories">
+              <Link to="/categories">Categories</Link>
+            </li>
           </ul>
```

Comment:
- [ ] Kindly separate the logic of the navigation bar from the Routes, this will make your code structured and easy to understand.

Reply:
hey @andersonlebon you mean i create a new ne file for routes